benchmarks/time_cosmology.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+from astropy import units as u
+from astropy.cosmology import FlatLambdaCDM, LambdaCDM
+
+
+class LambdaCDMBenchmarks:
+    H0 = 65 * u.km / u.s / u.Mpc
+    TCMB0 = 2.7 * u.K
+    params = [
+        LambdaCDM(H0, 0.25, 0.65, TCMB0, 3.04),
+        LambdaCDM(H0, 0.6, 0.7, TCMB0, 4),
+        LambdaCDM(H0, 0.4, 0.2, TCMB0, 3.04),
+        FlatLambdaCDM(H0, 0.25, TCMB0, 3.04),
+        FlatLambdaCDM(H0, 0.25, TCMB0, 3.04,
+                      [0.05, 0.1, 0.15] * u.eV)
+    ]
+
+    def setup(self, cosmo):
+        self.cosmology = cosmo
+        self.test_zs = np.linspace(0.1, 5.0, 200)
+
+    def teardown(self, cosmo):
+        del self.cosmology
+        del self.test_zs
+
+    def test_lumdist(self, cosmo):
+        self.cosmology.luminosity_distance(self.test_zs)
+
+    def test_age(self, cosmo):
```

Comment:
@aconley - I think this is all good except that ``test_`` should be ``time_``, here and above, otherwise the benchmarks don't get picked up